domains/index.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: steps
+title: Domains
+has_children: false
+permalink: /domains/
+image: false
+title: "Domains"
+subhead: "Manage all custom domains on your site"
+context:
+  - title: Getting to the Domains Homepage
+    steps:
+
+    - instruction: Login to the Dashboard
+      image: '/assets/images/screenshots/draft2/login.png'
+
+    - instruction: Click the Side Bar menu button to the left of the page
+      image: /assets/images/mockdown/draft2/domains/open-the-sidebar.png
+
+    - instruction: Locate the Settings dropdown menu inside the side bar menu to the left of the page
+      notes:
+      - header: Click on the link to expand the menu
+      image: /assets/images/mockdown/draft2/domains/sidebar-retracted.png
+
+    - instruction: Click the Stores link in the drop down
+      image: /assets/images/mockdown/draft2/domains/sidebar-expanded.png
+
+    - instruction: This takes you to a list of your stores
+      notes:
+      - header: Click the "Domains" button for the store you would like to add your custom domain to
+      image: /assets/images/mockdown/draft2/domains/stores-index.png
+
+    - instruction: This is the Domains Homepage. Here you will find all the information you need to maintain the domains tied to your store.
```

Comment:
This is the only `instruction` that has a period at the end, so I would delete it to keep things consistent.